src/components/layout/menu/MainMenu.js

```diff
@@ -17,6 +17,7 @@ const articleTypeList = [
 ];
 
 const imageUrl = "https://scontent-nrt1-1.xx.fbcdn.net/v/t1.6435-9/118343719_1758645397608074_5920063687015986170_n.jpg?_nc_cat=101&ccb=1-7&_nc_sid=8bfeb9&_nc_ohc=nK2z7ttVorsAX9-huTi&_nc_ht=scontent-nrt1-1.xx&oh=00_AfB_2xh-jR3_Z06084BQGYrbhEEy37s8ox4ehG0JFe0r9w&oe=63A19AAF";
+const planToRradCounterWithSubscriprion = "0";
```

Comment:
"planToRradCounterWithSubscriprion" to be changed to "planToReadCounterWithSubscription"